### PR TITLE
Only include current pings for a metric (fixes #354)

### LINKED ISSUE #354: Glean Dictionary incorrectly stating historical data
[This probe](https://dictionary.protosaur.dev/apps/fenix/metrics/browser_search_with_ads) is an example problem. It indicates that it is sent in two pings: baseline and metrics. However the [probe-info-service](https://probeinfo.telemetry.mozilla.org/glean/fenix/metrics) tells a different story, that it was only sent on the baseline ping in the spring of 2020, which means that now almost _no_ baseline pings would include it (only those from very old clients).

@wlach suggested that we should be matching the definition to versions, which would require use of the `git-commits` (but only for a linear history, I believe).

### PR DESCRIPTION
**Pull request by wlach**
### Pull Request checklist

- [x] The pull request has a descriptive title (and a reference to an issue it
      fixes, if applicable)
- [x] All tests and linter checks are passing
- [x] The pull request is free of merge conflicts

### CODE REVIEW

File: scripts/glean.py [hunk 1]
```diff
@@ -49,11 +49,7 @@ def __init__(self, identifier: str, definition: dict, *, ping_names: List[str] =
         self._set_definition(definition)
         self._set_description(self.definition)
 
-        defn_pings = set(
-            [p for d in definition[self.HISTORY_KEY] for p in d.get("send_in_pings", ["metrics"])]
-        )
```
Comment (wlach) [inline scripts/glean.py:54]: I am pretty sure this type of logic is unnecessary-- by the time probe-scraper outputs the files for us to consume, the defaults should already be set:

https://github.com/mozilla/glean_parser/blob/f9b676646f82d93ca2698540646a6d0bd29ab3ad/glean_parser/parser.py#L320

**Review by fbertsch — APPROVED** [review]
LGTM - we could keep a separate issue open with a per-version system. That seems like a larger design overhaul that affects the glean-dictionary in multiple places

**Comment by wlach** [comment]
> LGTM - we could keep a separate issue open with a per-version system. That seems like a larger design overhaul that affects the glean-dictionary in multiple places

Yeah I really don't know what to do there. The probe dictionary shows the full json definition of what's in probe scraper, but that seems pretty half-baked to me and I'm not sure if that's much better than providing a link to the source (which we'll be doing in #223)

**Comment by fbertsch** [comment]
> Yeah I really don't know what to do there. The probe dictionary shows the full json definition of what's in probe scraper, but that seems pretty half-baked to me and I'm not sure if that's much better than providing a link to the source (which we'll be doing in #223)

It is definitely half-baked. That was my idea because it became confusing trying to figure out what the probe dictionary was doing, and is certainly only useful to data engineers who understand the probe-scraper.

**Pull request merged**